internal/cli/mongocli/clusters/onlinearchive/start.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package onlinearchive
```

Comment:
onlinearchive?

Reply:
Not sure I understand the question

Reply:
the PR seems to hint at monitoring, not sure why this new file is here about online archive?

Reply:
It was in one of the earlier commits but looks like it might be a follow-up/maybe a miss from https://github.com/mongodb/mongodb-atlas-cli/pull/1813

Reply:
I checked out the branch, this file is unused, can you please delete it or use it?